src/admin-views/settings/getting-started.php

```diff
@@ -0,0 +1,52 @@
+<?php
+/**
+ * Getting started banner section.
+ *
+ * @since TBD
+ *
+ * @var bool $etp_enabled         Event Tickets Plus enabled or not.
+ * @var array $et_resource_links  Knowledgebase links for Event Tickets.
+ * @var array $etp_resource_links Knowledgebase links for Event Tickets.
+ */
+
+$help_text = $etp_enabled
+	? __( 'Thank you for using Event Tickets and Event Tickets Plus! We recommend looking through the settings below so that you can fine tune your specific ticketing needs. Here are some resources that can help.', 'event-tickets' )
+	: __( 'Thank you for using Event Tickets! We recommend looking through the settings below so that you can fine tune your specific ticketing needs. Here are some resources that can help.', 'event-tickets' );
+
+?>
+<div id="event-tickets__admin-banner">
```

Comment:
Is there any specific reason why you're using and ID here? I'd use class directly.

Reply:
No specific reason, moving out as a class.